src/native/libs/System.Security.Cryptography.Native.Android/pal_trust_manager.c

```diff
@@ -0,0 +1,75 @@
+#include "pal_trust_manager.h"
+
+static RemoteCertificateValidationCallback verifyRemoteCertificate;
+
+void AndroidCryptoNative_RegisterRemoteCertificateValidationCallback(RemoteCertificateValidationCallback callback)
+{
+    verifyRemoteCertificate = callback;
+}
+
+jobjectArray InitTrustManagersWithDotnetProxy(JNIEnv* env, intptr_t sslStreamProxyHandle)
```

Comment:
Might be good to add a comment about what exactly this is doing / returning, as I don't think its expected behaviour/usage is obvious from the name.

Reply:
That's a good point. I'll think about the naming and if I don't come up with a better name for the method, I'll add an explanation comment.

Reply:
I revisited the method and I realized that it's overcomplicated. There's no need to get the default trust managers and replace the X509TrustManager because we can simply create a custom array with just our trust manager.